src/components/common/Interactive.tsx

```diff
@@ -108,36 +101,26 @@ const InteractiveBase = ({ onMove, onKey, ...rest }: Props) => {
         left: keyCode === 39 ? 0.05 : keyCode === 37 ? -0.05 : 0,
         top: keyCode === 40 ? 0.05 : keyCode === 38 ? -0.05 : 0,
       });
-    },
-    [onKeyCallback]
-  );
-
-  const handleMoveEnd = useCallback(() => setDragging(false), []);
```

Comment:
This was a source of potential errors and memory leaks.

When the component is about to be unmounted while the user is dragging the mouse:

1) We call `useIsomorphicLayoutEffect` cleanup
2) It checks that `isDragging` is true. 
3) `toggleDocumentEvents` won't be called
4) Component gets unmounted
=> we have a memory leak

Then the user releases  the mouse:
1) We call `handleMoveEnd`
2) It tries to `setState`
3) The component is unmounted => we receive the error

With the new flow we prevent memory leak and these errors

Reply:
The thing that bothers me now is abandoned event listeners.

I mean if we do not call `toggleDocumentEvents` on the component unmount the `touchmove` and `touchend` listeners are still active if the picker was unmounted in the middle of the dragging.

For example, removing `touchmove` and `touchend` events after a document unmounting requires an extra click on the screen.

Reply:
In the previous implementation before the PR, we had the problem, that `toggleDocumentEvents` isn't called either. 
`isDragging` was set to true, so that the toggle cannot be executed. Which is the source of the memory leak. (after the component gets dismounted we are unable to change `isDragging` therefore we are unable to be unsubscribed)

The current solution doesn't call unsubscribe when the component dismounts, but as soon as the user generates a touch-up event we unsubscribe everything, plus all the handlers which were active don't make any effort, as we check containers (and they were false, as the component gets destroyed). 
If you think we want to unsubscribe at the time of components dismount, I can add useEffect with that. wdyt?

Reply:
Can't reproduce the memory leak and abandoned event listeners locally.

https://user-images.githubusercontent.com/206567/130655471-3d90f2d6-48d3-4434-ae45-ac5143cf2de7.mov

Reply:
Oh, it's my mistake, I had the wrong test 🤦 Sorry for that 

Talking about cleaning up, I added commit which covers that case